SudokuSolver/SudokuSolver/Common/Sudoku.cs

```diff
@@ -27,6 +27,53 @@ public Sudoku(int[,] grid)
             ValSqrt = (int)Math.Sqrt(RowsNumber);
         }
 
+        public int Get(int i, int j)
```

Comment:
Jeśli przeszkadzałoby ci na przyszłość to takie metody można zwinąć do tzw. "expression bodied method", i wtedy wyglądałoby to tak:
`public int Get(int i, int j) => Grid[i,j];`
ale nie jest to wymagane, ot syntax sugar